src/components/pages/Settings/Expansions/ActiveSets/ExpansionList/Checkbox.tsx

```diff
@@ -52,3 +52,9 @@ export default connect(
   mapStateToProps,
   mapDispatchToProps
 )(React.memo(Checkbox))
+
+function formatExpansionName(expansion: types.Expansion): string {
+  return expansion.wave
+    ? expansion.name + ' (' + expansion.wave + ')'
```

Comment:
@MilanTheHawk I would prefer a template literal here:

```ts
`${expansion.name} (${expansion.wave})`
```
But no need to change this. We can refactor this, when we stumble over it in the future ;)